d3d8to9.vcxproj

```diff
@@ -79,6 +79,7 @@
     <ClCompile Include="source\d3d8to9_vertex_buffer.cpp" />
     <ClCompile Include="source\d3d8to9_volume.cpp" />
     <ClCompile Include="source\d3d8types.cpp" />
+    <ClCompile Include="source\InterfaceQuery.cpp" />
```

Comment:
Please use the same naming convention as the other files for consistency: `interface_query.cpp`. =)

Reply:
Renamed the file to `interface_query.cpp`.